### ISSUE TITLE
Pets disappearing?

### ISSUE THREAD

**Issue by foxryk** [voidpet/bugs#581]
My starter pet, and at least one other, have completely disappeared from my game. I've checked the memory and it is NOT there, but I know for a fact I have not used the forget mechanic on it because it is my starter and I deeply care for it. It was a stage two anxious, I don't recall the exact level, and it was wearing a scarf if any of this is relevant? I've been playing on desktop (google chrome on laptop) for the past day or two. Genuinely distraught at the loss of my lil buddy :( I have counted and there is the correct amount of pets in the memory as displayed in the number to count them (22/100) but I know there should be about 25 in there. Not sure what I did to cause this but it seems like a bit of a major issue ^^"

**Comment by benawad** [comment]
The good news is your starter is in the database and not being shown in memory for some reason.

Try refreshing in the meantime while I work on a more permanent solution

**Comment by alex-cart** [comment]
The same thing happened to my starter sad pet. My inventory also reads "31 pets" but only 30 are showing in the Memory panel, it does seem to be a problem with it just not appearing.
I believe this occurred after I moved my starter out of my "main" inventory and into memory -- maybe that was what caused the bug.
Thank you @benawad for working on a solution, its a great game so far!

**Comment by alex-cart** [comment]
@foxryk Do you know if you moved that pet out of your "main pets" panel recently? Like, moved him into the memory log?

**Comment by foxryk** [comment]
@benawad hey! fully closing the tab and reopening it brought back the 3 missing pets and made my counter go back up to 25. I believe this may have happened because I was 'forgetting' some other pets in the panel at the time without previously expanding the page fully to show the pets near the bottom; although this is just a shot in the dark and I'm not certain this is the case. 
@alex-cart I hadn't moved him into the memory log recently, it was about 2 hours prior!

**Comment by foxryk** [comment]
@alex-cart yes i saw that button when the issue occured but it did NOT load in the missing pets, and the counter for how many pets were in the memory was wrong, so I think we were having two different problems!

**Comment by twiigbranch** [comment]
I have the same problem, except it's not my starter. Some of my pets have disappeared but still show up in my pet log, and some pets show up in my pet log which I haven't caught and don't own. Reloading doesn't help
